src/cv2pdb.vcxproj

```diff
@@ -34,7 +34,7 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'" Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
-    <PlatformToolset>v120_xp</PlatformToolset>
+    <PlatformToolset>$(DefaultPlatformToolset)</PlatformToolset>

```

Comment:
I don't think this works in VS2017, that is still used by the Appveyor build of Visual D that includes cv2pdb. Can you add defaulting DefaultPlatformToolset to v120_xp if it is not yet set?

Reply:
The reason I changed this is because the program no longer compiles with the `v120` tools, since those don't support `constexpr`

Reply:
Hmm, this is supposed to work according to https://github.com/rainers/cv2pdb/pull/69#discussion_r762519015 , maybe depends on the exact version of the compiler.
It builds on appveyor with cl 14.16.27023: https://ci.appveyor.com/project/rainers/visuald/builds/42205296

Adding something like
```
<DefaultPlatformToolset Condition="'$(DefaultPlatformToolset)' == ''">v120_xp</DefaultPlatformToolset>
```

to the property group a couple of lines above should do the trick to use the default on newer VS versions.

Reply:
It builds because the Makefile explicitly overrides `PlatformToolset` to `v141`, which is Visual Studio 2017: https://github.com/dlang/visuald/blob/9c90ff0e9f1e93962324486375426892665926d9/Makefile#L170

`v120` is Visual Studio 2013, [which doesn't support `constexpr`](https://docs.microsoft.com/en-us/previous-versions/hh567368(v=vs.140)?redirectedfrom=MSDN#corelanguagetable), and was disabled in 2019: https://github.com/dlang/visuald/commit/15216ca7454bed672daec9b14633da4eb99d5dd5

Reply:
Oops, indeed. Sorry for the noise. So yes. let's get rid of `v120_xp`.